__tests__/heroku.test.ts

```diff
@@ -0,0 +1,93 @@
+import { Polly } from '../test/polly';
+import { handleError, HerokuClient } from '../src/heroku';
+
+let polly: Polly;
+
+jest.useFakeTimers();
+
+beforeEach(() => {
+  polly = new Polly('api.heroku.com', {
+    adapters: ['node-http'],
+  });
```

Comment:
There's some recording setup code in the SDK now that you can use to get rid of this and your polly file but that can come in a later PR.

Reply:
Thanks - this is already removed in #3!